core-models/src/main/scala/com/mesosphere/usi/core/models/PodStatus.scala

```diff
@@ -13,7 +13,9 @@ import org.apache.mesos.v1.{Protos => Mesos}
   * there is no specification.
   */
 case class PodStatus(id: PodId, taskStatuses: Map[TaskId, Mesos.TaskStatus]) {
+  private val activeTaskStatus = Set(Mesos.TaskState.TASK_RUNNING, Mesos.TaskState.TASK_STAGING)
```

Comment:
https://github.com/apache/mesos/blob/master/include/mesos/mesos.proto#L2342

should TASK_STARTING be included here?